src/main/kotlin/no/skatteetaten/aurora/boober/feature/AlertsFeature.kt

```diff
@@ -71,7 +76,7 @@ class AlertsFeature : Feature {
                     AuroraConfigFieldHandler("$featureName/$name/description")
                 )
             }.toSet()
-        return definedAlerts
+        return definedAlerts.union(alertsDefaults)
```

Comment:
Her kan du slå sammen begge set-listene med +.

```suggestion
        return definedAlerts + alertsDefaults
```